src/Proto3/Suite/JSONPB/Class.hs

```diff
@@ -103,11 +105,22 @@ class ToJSONPB a where
   toEncodingPB :: a -> Options -> A.Encoding
   toEncodingPB x = A.toEncoding . toJSONPB x
 
+instance ToJSONPB A.Value where
+  toJSONPB v _ = v
+  toEncodingPB v _ = E.value v
+
+instance ToJSONPB A.Encoding where
+  toJSONPB e _ = fromJust . A.decode . Builder.toLazyByteString . E.fromEncoding $ e
```

Comment:
`fromJust` is not safe and will throw an exception if `A.decode` produces a `Nothing`. What is the rationale behind this behavior and can we explain that rationale in a comment?